src/components/EmissionsViewTable/EmissionsViewTable.js

```diff
@@ -38,7 +38,7 @@ export const EmissionsViewTable = ({ monitorPlanId, filterApply, setFilterApply
         else
         {
             setViewData(reduxCurrentTab?.viewData || []);
-            setMessage('Select a Reporting Period, Location, and Template and Apply Filter to view the data');
+            setMessage('Select a Reporting Period, Location, and Template, then click the Apply Filter button to view the data.');
```

Comment:
For consistency, I'd say either add the trailing period everywhere or leave it off. The initial message does **not** have a trailing period.